### ISSUE TITLE
`ls *` makes TypeStream hang and then throw a `Java heap space` error.

### ISSUE THREAD

**Issue by krisajenkins** [typestreamio/typestream#51]
I installed TypeStream:

```
$ typestream local start
...
$ typestream local seed
...
```

...then ran the CLI and started exploring:

```
$ typestream           
/ > cd dev/kafka/local
/dev/kafka/local
/dev/kafka/local > ls
brokers
consumer-groups
schemas
topics
```

...so far, so good. Now I tried `ls *`, which seems like a reasonable UNIX-y thing to do. I'd expect it to return something similar to `ls` in this case, but instead it hangs for a minute then errors out:

```
/dev/kafka/local > ls *

^C^C^C^C^C^CJava heap space
```

Checking around, it looks like `ls *` fails regardless of directory. 

FTR, I'm using OSX 13 and:

```
$ typestream --version
typestream version 2023.10.18+1 8b4e1125cd5f8dacf90ee147327e7541a625df94
$ java -version
openjdk version "19.0.1" 2022-10-18
OpenJDK Runtime Environment Zulu19.30+11-CA (build 19.0.1+10)
OpenJDK 64-Bit Server VM Zulu19.30+11-CA (build 19.0.1+10, mixed mode, sharing)
```

**Comment by lucapette** [comment]
hey @krisajenkins no I idea how missed this! 

First of all, thank you for the outstanding bug report 👏

It's definitely a bug. TypeStream doesn't really support the `*` syntax (yet?) but surely the parser shouldn't go into an infinite loop.

I think I caught this a few days ago and a fix is already in main. But I'll leave it open until I can confirm that's the case.

**Comment by lucapette** [comment]
I verified that this was indeed fixed in the last release (see #47 which seems unrelated but it's in fact the core problem behind the infinite loop we had 🤦 ). 

Not very happy about the fact that `*` doesn't work and while I don't really want to support it (as I don't see any specific use case that _must_ use `*`) I'm pretty sure people just expect it to work due the nature of the interaction being so similar to UNIX. I took some notes about this and will add `*` support to our roadmap asap.
